theme/templates/accounts/profile.html

```diff
@@ -358,6 +358,73 @@ <h5><span class="text-muted">Date: </span>{{ res.created }}</h5>
                             </div>
 
                             <div class="col-md-5">
+                                {#        Quota      #}
+                                {% if not profile_user.is_superuser %}
+                                    {% if profile_user.id == user.id or user.is_superuser %}
+                                        <div class="panel panel-default">
+                                            <div class="panel-heading">
+                                                <h3 class="panel-title">
+                                                    <span class="fa fa-database" style="margin-right: 1.2rem">&nbsp;</span>Quota
+                                                </h3>
+                                            </div>
+                                    
+                                            <div class="panel-body">
+                                                {% with quota_data as qd %}
+                                                <div class="quota">
+                                                    {% if qd.remaining == 0 %}
+                                                        <div class="progress">
+                                                            <div class="progress-bar progress-bar-danger" 
+                                                            style="width: {%if qd.percent_over > 100%}100{%else%}{{qd.percent_over}}{%endif%}%"
+                                                                title="{{ qd.percent_over|floatformat:"1" }}% over allocated space">
+                                                                <span class="sr-only">{{ qd.percent_over|floatformat:"1" }}% over allocated space</span>
+                                                            </div>
+                                                        </div>
+                                                        <span class="text-danger">You are over your allocated quota by {{ qd.percent_over|floatformat:"1" }}%</span>
+                                                    {% else %}
+                                                        <div class="progress">
+                                                            <div class="progress-bar progress-bar-success" 
```

Comment:
I think we could use color thresholds as it is commonly used to represent use of allocated storage.

Perhaps:
Green from 0% to 69%
Orange from 70% to 89%
Red from 90% to 100%

Colors are listed here: https://getbootstrap.com/docs/3.3/components/#progress-alternatives

Reply:
I like that idea. It's a bit tricky because there are 2 progress bars here: one for the dataZone and one for the userZone. It is their sum that determines the total percent used. It doesn't really make sense to split the allocation in the middle, because each user migh use more datazone space vs userzone space.

Example: we alocate 20GB for devin. He likes to store things in hydroshare but never uses the datazone. Let's say I upload 15GB to the dataZone, I've used of 75% of my total allocation. But does it really make sense to show `Orange` for both of these progress bars, when one has 0% and one has 75%? If devin uploads 5GB more and has used 100% total allocated with 20GB in dataZone and 0GB in userZone, does it really make sense to show both progress bars in red even though one would show 0% and still be in red?

Reply:
Understood. In that case we can discard this idea.

Reply:
I really like the idea thanks Mauriel. I thought about using [stacked](https://getbootstrap.com/docs/3.3/components/#progress-stacked) bars with one [striped](https://getbootstrap.com/docs/3.3/components/#progress-striped) and one not in order to differentiate. Then we could use the colors as you suggested. But HS has BS3.0.2 and striped progress bars weren't added until ~BS3.3 :(

Reply:
I went ahead and _ROUGHLY_ implemented part of this idea.

Three screenshots to show the progression (blue/green = under 90% usage, orange/red = 90-100%, red = over 100%):
![image](https://github.com/hydroshare/hydroshare/assets/17934193/2b1c38e6-c65f-4445-867b-c9e613238662)
![image](https://github.com/hydroshare/hydroshare/assets/17934193/b92fa412-b82e-406a-9d3e-0f1db05d8e04)
![image](https://github.com/hydroshare/hydroshare/assets/17934193/b36f6dbe-1537-4f45-8939-b07e2b3c3ba6)

Reply:
This is better, but still a little unintuitive. We would need tooltip text for each bar indicating what storage it represents.

Would it be better to use two different progress bars (one above and one below) with their own breakdown and color coding?

Reply:
Current version has tooltips already.

But I see your point. I can move it to two separate bars instead of stacked.

Reply:
I'm going to upgrade our BS version so that I can use some `striped` bars. Back with more info once I complete that

Reply:
Okay with the addition of stripes, it looks like:
### 75% of allocated quota used in DataZone, 5% used in UserZone:
![image](https://github.com/hydroshare/hydroshare/assets/17934193/06849e08-3228-416d-a898-125c5662234d)

### 45% used in DataZone, 50% in UserZone:
![image](https://github.com/hydroshare/hydroshare/assets/17934193/b26b2560-8aa8-429e-bbde-fbb83e27ebf6)

### 25% use in both zones
![image](https://github.com/hydroshare/hydroshare/assets/17934193/4c5a78bc-ec7a-4360-abac-eac1934b11b0)